parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -632,6 +686,7 @@ construct_runtime!(
 		// The main stage.
 		Assets: pallet_assets::<Instance1>::{Pallet, Call, Storage, Event<T>} = 50,
 		Uniques: pallet_uniques::{Pallet, Call, Storage, Event<T>} = 51,
```

Comment:
so `Uniques: pallet_uniques` stays or any migration to nfts?

Reply:
It's going to stay for now until we get the UI support for the new pallet and make a decision on the proper migration path